src/pybind/mgr/cephadm/upgrade.py

```diff
@@ -177,7 +178,9 @@ def _wait_for_ok_to_stop(self, s: DaemonDescription) -> bool:
             if not self.upgrade_state or self.upgrade_state.paused:
                 return False
 
-            r = self.mgr.cephadm_services[s.daemon_type].ok_to_stop([s.daemon_id])
+            # setting force flag to retain old functionality. Should revisit.
```

Comment:
If we set `force=False` for small clusters, we might block the upgrade due to not enough grafana instances. I think keeping `force=True` here is the right thing to do.